packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -844,6 +848,31 @@ export class StripeHelper {
     return !!subscription;
   }
 
+  /**
+   * Adds the appropriate tax id if found to the customer based on passed in
+   * currency or the customers existing currency.
+   **/
+  async addTaxIdToCustomer(customer: Stripe.Customer, currency?: string) {
+    const taxId =
+      this.taxIds[
+        currency?.toUpperCase() ?? customer.currency?.toUpperCase() ?? ''
+      ];
+    if (taxId) {
+      await this.stripe.customers.update(customer.id, {
+        invoice_settings: { custom_fields: [{ name: 'Tax ID', value: taxId }] },
+      });
+    }
+  }
+
+  /**
+   * Returns the customers tax id if they have one.
+   **/
+  customerTaxId(customer: Stripe.Customer) {
+    return customer.invoice_settings.custom_fields?.find(
+      (field) => field.name === 'Tax ID'
+    );
+  }
+
```

Comment:
The two functions added here don't seem to have tests?

Reply:
They do now! 😁